docs/troubleshooting/report-scylla-problem.rst

```diff
@@ -132,7 +151,7 @@ There are several commands you can use to see if there is a performance issue on
 Use ``scyllatop`` instead. 
 
 .. note:: 
-   To help the ScyllaDB support team assess your problem, it is best to pipe the results to a file which you can attach with the Health Check report.
+   To help the ScyllaDB support team assess your problem, it is best to pipe the results to a file which you can attach with Scylla Doctor Vitals.
```

Comment:
Again, not "Scylla Doctor Vitals" - Vitals are just a json file. SD collects Vitals **and** logs.